training-manager.lic

```diff
@@ -22,9 +22,7 @@ class TrainingManager
     town_data = get_data('town')
     @hometown = town_data[@settings.hometown]
     @skip_repair = @settings.skip_repair
-	@sell_loot = @settings.sell_loot
-    @repair_timer = @settings.repair_timer
-    UserVars.repair_timer_snap ||= Time.now
+    @sell_loot = @settings.sell_loot
```

Comment:
* Fix indent on line 25
* Moved `@repair_timer` and `UserVars.repair_timer_snap` logic to `crossing-repair` script